### ISSUE TITLE
User deletion broken

### ISSUE THREAD

**Issue by troeger** [troeger/kubeportal#12]
On a production installation in Kubernetes 1.15, user deletion leads to a HTTP 500 error because of a missing table. This is surprising, since the startup did no migrations:

```
~# kubectl -n kubeportal log kubeportal-9cf6fdf44-56vtk
Operations to perform:
  Apply all migrations: admin, auth, contenttypes, kubeportal, oauth2_provider, oidc_provider, sessions, social_django
Running migrations:
  No migrations to apply.
Superuser exists, nothing changed.
[uwsgi-static] added mapping for /static => /code/static-collected
*** Starting uWSGI 2.0.18 (64bit) on [Wed Oct  9 10:21:15 2019] ***
compiled with version: 8.3.0 on 13 August 2019 10:11:39
os: Linux-4.15.0-64-generic #73-Ubuntu SMP Thu Sep 12 13:16:13 UTC 2019
nodename: kubeportal-9cf6fdf44-56vtk
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 64
current working directory: /
detected binary path: /usr/local/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
!!! no /etc/mime.types file found !!!
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 0.0.0.0:8000 fd 4
uwsgi socket 0 bound to TCP address 127.0.0.1:36249 (port auto-assigned) fd 3
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.7 (default, Jul 13 2019, 15:53:53)  [GCC 8.3.0]
Python main interpreter initialized at 0x561d28c003a0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 208576 bytes (203 KB) for 4 cores
*** Operational MODE: threaded ***
added /code/ to pythonpath.
WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0x561d28c003a0 pid: 68 (default app)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 68)
spawned uWSGI worker 1 (pid: 84, cores: 4)
spawned uWSGI http 1 (pid: 88)
...
Allowing authorization through subrequest for user ptroeger with service account 'default:root'.
[pid: 84|app: 0|req: 9/9] 10.244.103.192 () {64 vars in 1320 bytes} [Wed Oct  9 10:21:30 2019] GET /subauthreq => generated 0 bytes in 161 msecs (HTTP/1.1 200) 5 headers in 975 bytes (1 switches on core 0)
Internal Server Error: /admin/kubeportal/user/42/delete/
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/usr/local/lib/python3.5/site-packages/django/db/backends/sqlite3/base.py", line 383, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.OperationalError: no such table: oauth2_provider_application

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.5/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.5/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.5/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.5/site-packages/django/contrib/admin/options.py", line 606, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "/usr/local/lib/python3.5/site-packages/django/utils/decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/usr/local/lib/python3.5/site-packages/django/views/decorators/cache.py", line 44, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/usr/local/lib/python3.5/site-packages/django/contrib/admin/sites.py", line 223, in inner
    return view(request, *args, **kwargs)
  File "/usr/local/lib/python3.5/site-packages/django/utils/decorators.py", line 45, in _wrapper
    return bound_method(*args, **kwargs)
  File "/usr/local/lib/python3.5/site-packages/django/utils/decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/usr/local/lib/python3.5/site-packages/django/contrib/admin/options.py", line 1825, in delete_view
    return self._delete_view(request, object_id, extra_context)
  File "/usr/local/lib/python3.5/site-packages/django/contrib/admin/options.py", line 1846, in _delete_view
    deleted_objects, model_count, perms_needed, protected = self.get_deleted_objects([obj], request)
  File "/usr/local/lib/python3.5/site-packages/django/contrib/admin/options.py", line 1820, in get_deleted_objects
    return get_deleted_objects(objs, request, self.admin_site)
  File "/usr/local/lib/python3.5/site-packages/django/contrib/admin/utils.py", line 118, in get_deleted_objects
    collector.collect(objs)
  File "/usr/local/lib/python3.5/site-packages/django/contrib/admin/utils.py", line 181, in collect
    return super().collect(objs, source_attr=source_attr, **kwargs)
  File "/usr/local/lib/python3.5/site-packages/django/db/models/deletion.py", line 223, in collect
    elif sub_objs:
  File "/usr/local/lib/python3.5/site-packages/django/db/models/query.py", line 278, in __bool__
    self._fetch_all()
  File "/usr/local/lib/python3.5/site-packages/django/db/models/query.py", line 1242, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/usr/local/lib/python3.5/site-packages/django/db/models/query.py", line 55, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "/usr/local/lib/python3.5/site-packages/django/db/models/sql/compiler.py", line 1100, in execute_sql
    cursor.execute(sql, params)
  File "/usr/local/lib/python3.5/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/usr/local/lib/python3.5/site-packages/django/db/backends/utils.py", line 76, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/usr/local/lib/python3.5/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/usr/local/lib/python3.5/site-packages/django/db/utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/usr/local/lib/python3.5/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/usr/local/lib/python3.5/site-packages/django/db/backends/sqlite3/base.py", line 383, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: no such table: oauth2_provider_application
```
This might be related to a problem already identified by @leonardlorenz with Docker Compose and local development.

**Comment by leonardlorenz** [comment]
The problem seems to be that kubeportal does in fact write to the correct database, but doesn't read from the correct one.

There exist 2 databases in my container for some reason. 
- /code/db.sqlite3 [0 bytes]
- /data/kubeportal.sqlite3 [416 kbytes]

So migrations can work perfectly fine but it will still act as if the migrations never happened. I'll keep investigating.

**Comment by leonardlorenz** [comment]
I figured out the database problem. It was a fairly obvious mistake on my side.
Originally this was what the new docker-entry.sh entries looked like
```Bash
/code/manage.py createsuperuser --username $API_USER_USERNAME --email $API_USER_EMAIL --noinput
/code/manage.py drf_create_token $API_USER_USERNAME
```
I now noticed that both of them lacked the configuration flag. The following snippet fixes it.
```Bash
/code/manage.py createsuperuser --username $API_USER_USERNAME --email $API_USER_EMAIL --noinput --configuration=Production
/code/manage.py drf_create_token $API_USER_USERNAME --configuration=Production
```

With that embarrassing fact said, I don't think the problem you stated in the original issue is related. I'll investigate it when I'm done with the other tasks.

**Comment by troeger** [comment]
Still in existence with v0.2.0

**Comment by leonardlorenz** [comment]
Seems to work fine now, please test again.

**Comment by troeger** [comment]
Fixed with v0.2.2
